### PR TITLE
83 playlist modify implication

### PR DESCRIPTION
**Pull request by dqieu**
- finish playlist modify ABR 
- CHANGE: Playlist to handle ArrayList<Song> as contents to Arraylist<String> (SongID)
- Controllers for all functionality completed
- DTO for all functionality completed to ensure Clean Arch, CHANGE: PlaylistModifyResponseModel + PlaylistCreateResponseModel = PlaylistReponseModel
- PENDING (thanks in advance, Alex): implement buttons/UX for Playlist's controllers, and update ViewModel given DTO.

### CODE REVIEW

**Review by fvkxvsop — APPROVED** [review]
looks bless, approving so I can use this stuff to clean up ViewModels for Record and Queue

File: src/main/java/interface_adaptors/playlist_ia/PlaylistModifyControl.java [hunk 1]
```diff
@@ -1,4 +1,44 @@
 package interface_adaptors.playlist_ia;
 
+import abr.playlist_abr.PlaylistModifyInputBoundary;
+import abr.playlist_abr.PlaylistModifyRequestModel;
+import entities.playlist_entities.Privacy;
```
Comment (Starliyiyu) [inline src/main/java/interface_adaptors/playlist_ia/PlaylistModifyControl.java:5]: entities imported to ia layer

**Review by Starliyiyu — APPROVED** [review]
minor dependency issues (see comment on code), otherwise looks good

**Review by 2368883029 — APPROVED** [review]
Looks promising, with minor Clean Architecture violation mentioned above

**Pull request merged**